extension/templates/devTools/activity/graph.js

```diff
@@ -0,0 +1,90 @@
+define(['d3', 'util/Radio'], function(d3, Radio) {
```

Comment:
curious what you think of this, but i could see it in `extension/js/inspector/app/modules/Activity/d3/graph`. Why? it's a js module and happens to primarily be D3... lol